hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -21,10 +21,10 @@
 import org.apache.hudi.common.model.HoodieRecord;
 import org.apache.hudi.common.testutils.SchemaTestUtil;
 import org.apache.hudi.common.util.Option;
-import org.apache.hudi.sync.common.AbstractSyncHoodieClient.PartitionEvent;
```

Comment:
Can we avoid these changes assuming they are unintended ?

Reply:
I did the usual reformatting from within intellij. I didn't explicitly fix any of these.